Unix/etc/omiserver.conf

```diff
@@ -1,4 +1,4 @@
-# omiserver configuration file
+# omiserver configuration file, more detail: https://github.com/Microsoft/omi/#configuring-omi-server
```

Comment:
This is a `.conf` file, restrict lines to 80 bytes maximum please. Didn't I mention that in last review?